src/cnxml/serialize.js

```diff
@@ -132,7 +140,22 @@ function list(obj, children) {
  * Serializer for cross-references.
  */
 function xref(obj, children) {
-    return <link target-id={obj.data.get('target')}>
+    let attrs = {
+        'target-id': obj.data.get('target')
+    }
+
+    const declension = obj.data.get('declension')
+    const cmlnleCase = declension ?
+        DECLENSIONS.some(dec => dec === declension) ?
+            declension
+            : 'INVALID'
+        : null
```

Comment:
Why do you have this check here? You are guaranteed that the document you are serializing is correct according to its schema, and if you want to limit values of `cmlnle:case` then you should do it in schema.

Reply:
I was thinking this way:
schema should allow any string as `case`, but when serializing to cnxml we have to be sure that it is supported.

Reply:
This makes no sense. Why would you allow invalid values anywhere?

Reply:
In case when we want to serialize to another format which may be supporting those cases